ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/TableSource.java

```diff
@@ -48,23 +46,22 @@ public TableSource(
       @JsonProperty(value = "formats", required = true) final Formats formats,
       @JsonProperty("timestampColumn") final Optional<TimestampColumn> timestampColumn,
       @JsonProperty(value = "sourceSchema", required = true) final LogicalSchema sourceSchema,
-      @JsonProperty("pseudoColumnVersion") final OptionalInt pseudoColumnVersion,
-      @JsonProperty("stateStoreVersion") final Formats stateStoreVersion
+      @JsonProperty("pseudoColumnVersion") final int pseudoColumnVersion,
+      @JsonProperty("stateStoreFormats") final Formats stateStoreFormats
```

Comment:
The changes in this file should be accompanied by changes in schema.json (the one associated with KsqlPlanSchemaTest.json), no?